test/dlc_tests/container_tests/bin/testTensorBoard

```diff
@@ -1,9 +1,16 @@
 #!/bin/bash
-
+  
 HOME_DIR=/test
 BIN_DIR=${HOME_DIR}/bin
-LOG_DIR=${HOME_DIR}/logs
+LOG_DIR=${HOME_DIR}/logs/train/plugins/profile
```

Comment:
Could it be ```/logs/train/plugins/profile```? I did some local tests and I might be wrong

Reply:
log dir should be {scirpt_path}/logs, not /logs.

Reply:
The test script should be located in ```$BIN_DIR``` but not ```$HOME```.
Please also take a look of the test since it should pass on CPU images but failed.

I also tried to reproduce a successful run on functionality well images (e.g. 2.0.2 image or 2.2.0 CUDA10.1 image) and it seems to be ```/logs/``` but I still could not find a *.pb file.
The structure in my test is
```
/logs/train/plugins/profile/2020-06-26_19-35-07/local.trace
```